packages/terra-framework-docs/src/terra-dev-site/test/menu/menu/LargeMenuAutoWidth.test.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import Menu from 'terra-menu';
+import Button from 'terra-button';
+import IconCaretDown from 'terra-icon/lib/icon/IconCaretDown';
+import IconPrinter from 'terra-icon/lib/icon/IconPrinter';
+
+class LargeMenuAutoWidth extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleButtonClick = this.handleButtonClick.bind(this);
+    this.handleRequestClose = this.handleRequestClose.bind(this);
+    this.setButtonNode = this.setButtonNode.bind(this);
+    this.getButtonNode = this.getButtonNode.bind(this);
+    this.state = { open: false };
+  }
+
+  componentDidMount() {
+    this.forceUpdate();
+  }
+
+  handleButtonClick() {
+    this.setState({ open: true });
+  }
+
+  handleRequestClose() {
+    this.setState({ open: false });
+  }
+
+  setButtonNode(node) {
+    this.buttonNode = node;
+  }
+
+  getButtonNode() {
+    return this.buttonNode;
+  }
+
+  render() {
+    return (
+      <div>
+        <div>
+          This menu should have a large height. And all items should be visible without scrolling.
+        </div>
+        <Menu
+          isOpen={this.state.open}
+          targetRef={this.getButtonNode}
+          onRequestClose={this.handleRequestClose}
+          contentWidth="auto"
```

Comment:
I'm seeing that this example works fine when added to main branch as well. Are we sure this is the right example to test the change?
<img width="1680" alt="image" src="https://github.com/cerner/terra-framework/assets/40137430/9d791226-67e4-4d2d-9417-edda98587fd7">

Reply:
Yes ,To recreate the scenario we can revert the changes, the given example wont work and instead will throw an broken menu .

Reply:
The above screenshot is from 'main'.  Example is not broken here

Reply:
Ok In this particular case you can increase the size of menu by adding the more menuitems .(When the menu size is smaller than screen size in that scenario it is working fine .But when screen size is smaller than menu size in that case we can recreate the broken menu)

Reply:
Does the reported issue happens when height is set auto as well ?

Reply:
No on height as auto it is working fine